src/store/entities.js

```diff
@@ -162,7 +162,18 @@ function loadingRejected(state, { payload }) {
 
 function selectEntity(state, { payload }) {
     const rows = [...state.rows];
-    const toSelect = [].concat(payload);
+    let toSelect = [];
+    if (Array.isArray(payload?.id)) {
```

Comment:
Here you use optional chaining, whereas in https://github.com/RedHatInsights/insights-inventory-frontend/pull/1860/files#diff-ac2c6e5c21bc579a2232094c424395566464d8ab26301d894dff7a4e1d7a30d6R39 it's strict. I think we better stick to the strict one already: if we never expect `payload` to be undefined, then we will have better means to catch and recognize an error